components/templates/ErrorTemplate/index.tsx

```diff
@@ -1,22 +1,31 @@
-import SignDirectionIcon from 'mdi-react/SignDirectionIcon'
-import { FunctionComponent } from 'react'
+import { ThemeContext } from '@hooks/contexts/theme'
+import Link from 'next/link'
+import { FunctionComponent, useContext } from 'react'
 
-import { StyledErrorPageWrapper, StyledErrorPageText } from './ErrorTemplateStyles'
+import {
+    StyledErrorPageWrapper,
+    StyledErrorPageText,
+    StyledErrorPageImageHeader,
+    StyledHeaderText,
+} from './ErrorTemplateStyles'
 
-const ErrorTemplate: FunctionComponent = () => (
-    <StyledErrorPageWrapper>
-        <div>
-            <div>
-                <SignDirectionIcon />
-            </div>
-        </div>
-        <h1>404: Not Found</h1>
-        <StyledErrorPageText>
-            The requested URL was not found. <br /> Return to{' '}
-            <a href="https://learn.sourcegraph.com">Sourcegraph Learn</a>
-        </StyledErrorPageText>
-    </StyledErrorPageWrapper>
+const ErrorTemplate: FunctionComponent = () => {
+    const theme = useContext(ThemeContext)
 
-)
+    return (
+        <StyledErrorPageWrapper>
+            <StyledErrorPageImageHeader>
+                <img alt='Sourcegraph robots' src='https://storage.googleapis.com/sourcegraph-assets/learn/images/sg_robots.svg' />
+            </StyledErrorPageImageHeader>
+            <StyledHeaderText>Not what you were ... searching for?</StyledHeaderText>
+            <StyledErrorPageText isDark={theme.isDark}>
+                Head back over to the {' '}
```

Comment:
Maybe "Return to Sourcegraph Learn."? I am wondering if "head back" is too idiomatic, but since this is playful I think you can keep it. I would do `Sourcegraph Learn` instead of `Learn homepage` and also add a full stop at the end as long as it doesn't look weird.

Reply:
Sounds good to me!